src/main/java/graphql/execution/CoercedVariables.java

```diff
@@ -0,0 +1,30 @@
+package graphql.execution;
+
+import graphql.Internal;
+import graphql.collect.ImmutableMapWithNullValues;
+
+import java.util.Map;
+
+/**
+ * Holds coerced variables
+ */
+@Internal
+public class CoercedVariables {
+    private final ImmutableMapWithNullValues<String, Object> coercedVariables;
+
+    public CoercedVariables(Map<String, Object> coercedVariables) {
+        this.coercedVariables = ImmutableMapWithNullValues.copyOf(coercedVariables);
+    }
+
+    public Map<String, Object> getMap() {
```

Comment:
map name it `toMap` that is you are converting it from a X -> Y - from a CoercedVariables to a map representtion